### DISCUSSION TITLE
Initial Suction for Unsaturated soil

### DISCUSSION THREAD

**Discussion by Rnsion** [Anura3D/Anura3D_OpenSource#70] [Q&A] [answered]
Dear all,

I am working on modelling slope in unsaturated condition and rainfall infiltrations. I am using unsaturated material with 2 phase suction effects with VG liquid retention curve and Mualem hydraulic conductivity curve. The material model that I use is Mohr Coulomb (Extended Mohr Coulomb not available). I also not modelling phreatic surface.

1. Is there possible I put initial suction without phreatic surface?
2. If I use unsaturated material - 3-phase fully coupled, is possible there is suction effect and how I put the the initial suction or degree of saturation in my model? 

Thank you all for any helps!
Rnsion

**Comment by FrancescaCeccato** [comment, marked as answer]
Dear Rnsion,
I'm not sure I got your question. Suction is a positive pressure and it will be distributed linearly with depth (y-coord). You can limit the maximum suction with the option max suction at soil surface of k0 procedure:

![image](https://user-images.githubusercontent.com/100557892/222157026-91463996-9b20-407a-8ff0-0501c60f55a0.png)

Your phreatic surface can be the bottom of the model.
If you want a constant suction you have to run a quasi-static step with suction at the boundary and waint for convergence. You will then remove it in the following step.
3phase works exactly the same.
I hope I was clear, maybe you can give us more detial about your problem.
